projects/storefrontapp/src/app/spartacus/spartacus-b2c-configuration.module.ts

```diff
@@ -17,6 +17,7 @@ import { environment } from '../../environments/environment';
 
 const defaultBaseSite = [
   'electronics-spa',
+  'electronics-spa-standalone',
```

Comment:
[MINOR]
It's not obvious why we need this basesite. Let's add a code comment e.g. `// used in the OPF integration`